BlockzillaPackage/Sources/Licenses/license-list.plist

```diff
@@ -0,0 +1,1224 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@razvanlitianu should [the copyright notice](https://github.com/mozilla-mobile/focus-ios/blob/main/Deferred/Deferred.swift#L2-L7) for Deferred also be included here?

Reply:
@jonalmeida Good catch. I was under the impression that Deferred is no longer used but there's still one place. I'll add the notice and then create a task to remove it.